packages/@atjson/document/src/annotation.ts

```diff
@@ -26,7 +46,7 @@ export default abstract class Annotation<Attributes = {}> {
       attributes: unprefix(this.vendorPrefix, this.subdocuments, attrs.attributes)
     });
   }
-
+  readonly vendorPrefix: string;
```

Comment:
👎  Please don't expose this as a public api; I'd prefer this to be accessed by accessing the constructor.

Reply:
Hmm.. 
I wasn't able to access it on the constructor, will take a look when I get home.

Reply:
It's usually being accessed via `this.constructor as typeof AnnotationConstructor<any, Attributes>`